eggplant_project/templates/base.html

```diff
@@ -21,6 +21,17 @@
 </head>
 <body class="{% block body_classes %}{% endblock %}">
 
+    {# This is the language chooser #}
```

Comment:
I think it could be cool to have this in a seperate template that gets included - no?